frontend/src/components/Playback/PlayerElement.vue

```diff
@@ -4,29 +4,27 @@
       :to="videoContainerRef"
       :disabled="!videoContainerRef"
       defer>
-      <div class="uno-my-auto">
-        <Component
-          :is="mediaElementType"
-          v-show="mediaElementType === 'video' && videoContainerRef"
-          ref="mediaElementRef"
-          :poster="String(posterUrl)"
-          autoplay
-          crossorigin
-          playsinline
-          :loop="playbackManager.isRepeatingOnce"
-          :class="{ 'uno-object-fill': playerElement.isStretched.value, 'uno-max-h-100vh': true}"
-          @loadeddata="onLoadedData">
-          <track
-            v-for="sub in playbackManager.currentItemVttParsedSubtitleTracks"
-            :key="`${playbackManager.currentSourceUrl}-${sub.srcIndex}`"
-            kind="subtitles"
-            :label="sub.label"
-            :srclang="sub.srcLang"
-            :src="sub.src">
-        </Component>
-        <SubtitleTrack
-          v-if="subtitleSettings.state.enabled && playerElement.currentExternalSubtitleTrack?.parsed !== undefined" />
-      </div>
+      <Component
+        :is="mediaElementType"
+        v-show="mediaElementType === 'video' && videoContainerRef"
+        ref="mediaElementRef"
+        :poster="String(posterUrl)"
+        autoplay
+        crossorigin
+        playsinline
+        :loop="playbackManager.isRepeatingOnce"
+        :class="{ 'uno-object-fill': playerElement.isStretched.value, 'uno-max-h-100vh': true}"
```

Comment:
Could you test the following style? I don't have a ton of media to test with myself. I believe this would fix the problem with video sizing and would allow us to keep the parent div element.

`:class="{ 'uno-object-cover': playerElement.isStretched.value, 'uno-w-screen': playerElement.isStretched.value, 'uno-h-full': true }"`

Reply:
I tested with uno-object-cover uno-w-screen uno-max-h-100vh. For fullscreen it kinda worked, however it does not limit height. This causes a narrow window to cut video in the top and bottom.

Without the object-fit it seems a lot better. I wonder if the video container should always try to be wide.

```
:class="{ 'uno-w-screen': true, 'uno-h-full': true }"
```

I still don't get why the stretch option is wanted. I don't know anyone that asked for the video to not be scaled to fit the screen

Reply:
As far as I can find in forums, i.e. https://forum.jellyfin.org/t-stretch-as-deafult-please,
stretch seems to have been a sort of inbetween option between not scaling up, and scaling up with some compromise. If this fix works I don't know that the stretch option will be needed anymore